lib/tasks/demo_seed.rake

```diff
@@ -298,12 +298,12 @@ namespace :demo do
     end
 
     # create a second nufsaccount for split accounts
-    nufsaccount2 = NufsAccount.find_by_account_number("111-2222222-44444444-01")
+    nufsaccount2 = NufsAccount.find_by_account_number("123-1234567-12345678-01")
```

Comment:
Good idea. Finder updates are overdue. I cleaned up this one along with the others Rubocop found.